core/src/main/java/jenkins/model/item_category/ItemCategory.java

```diff
@@ -0,0 +1,112 @@
+package jenkins.model.item_category;
+
+import hudson.Extension;
+import hudson.ExtensionPoint;
+import hudson.model.ModelObject;
+import jenkins.model.Messages;
+
+/**
+ * A category for {@link hudson.model.Item}s.
+ *
+ * @since TODO
+ */
+public abstract class ItemCategory implements ModelObject, ExtensionPoint {
+
+    public static int MIN_WEIGHT = 0;
+
+    public static int MIN_TOSHOW = 1;
+
+    /**
+     * Identifier, e.g. "category-id-standaloneprojects", etc.
```

Comment:
:ant: just an example, but does not match the actual value chosen for that category, which could be confusing.